pyproject.toml

```diff
@@ -51,6 +50,7 @@ typer = { version = "^0.12.3", optional = true }
 pytest = { version = "*", optional = true }
 pyyaml = { version = "^6", optional = true }
 eval-type-backport = { version = "^0.2.2", python = "~3.9" }
+dulwich = "^0.22.7"
```

Comment:
Do you know if there is a minimum version required for the feature that we are using ?
Best to keep the dependencies as loose as possible for the SDK